frontend/src/components/regions/UsersAndGroups/UsersAndGroups.test.js

```diff
@@ -72,3 +73,42 @@ test("cleanAndFilterTeams can deal with multiple words in any order", () => {
   const names = ret.map((group) => group.name);
   expect(names).toEqual(["demo group 1 AAA", "demo group 2 aaa"]);
 });
+
+test("getPrStatus should return a red text whenever an oldest pull request is older than two days", () => {
```

Comment:
Please dont hard code configuration into your tests.  We do not mix configuration and code. What if we decided to change the configuration? Your tests would need to be rewritten! We are very likely to change these values.

Make proper use of the constants you defined. 

PR_WARNING_AGE_THRESHOLD
PR_ERROR_AGE_THRESHOLD
TILDE_ERROR_AGE_THRESHOLD
TILDE_WARNING_AGE_THRESHOLD